pkg/webservice/dao/node_util.go

```diff
@@ -18,6 +18,12 @@
 
 package dao
 
+type PartitionNodesUtilDAOInfo struct {
+	ClusterId     string              `json:"clusterId,omitempty"`
+	Partition     string              `json:"partition,omitempty"`
+	NodesUtilList []*NodesUtilDAOInfo `json:"utilizationList,omitempty"`
```

Comment:
how about naming it `utilizations`?

Reply:
It sounds good to me, renamed 'utilizationList' to 'utilizations'